submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/rest/MetricRestApi.java

```diff
@@ -0,0 +1,181 @@
+import org.apache.submarine.server.workbench.database.entity.Metric;
+import org.apache.submarine.server.workbench.database.service.MetricService;
+import org.apache.submarine.server.response.JsonResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/metric")
+@Produces("application/json")
+@Singleton
+public class MetricRestApi {
+  private static final Logger LOG = LoggerFactory.getLogger(LoginRestApi.class);
+  MetricService metricService = new MetricService();
+
+  @Inject
+  public MetricRestApi() {
+  }
+
+  /*
+  # +-------+----------+--------------+---------------+------+--------+------------------+
+  # | key   | value    | worker_index | timestamp     | step | is_nan | job_name         |
+  # +-------+----------+--------------+---------------+------+--------+------------------+
+  # | score | 0.666667 | worker-1     | 1569139525097 |    0 |      0 | application_1234 |
+  # | score | 0.666667 | worker-1     | 1569149139731 |    0 |      0 | application_1234 |
+  # | score | 0.666667 | worker-1     | 1569169376482 |    0 |      0 | application_1234 |
+  # | score | 0.666667 | worker-1     | 1569236290721 |    0 |      0 | application_1234 |
+  # | score | 0.666667 | worker-1     | 1569236466722 |    0 |      0 | application_1234 |
+  # +-------+----------+--------------+---------------+------+--------+------------------+
+  */
+
+  @GET
+  @Path("/list")
+  @SubmarineApi
+  public Response listMetric(@QueryParam("metricKey") String metricKey,
+                              @QueryParam("value") Float value,
+                              @QueryParam("workerIndex") String workerIndex,
+                              @QueryParam("timestamp") BigInteger timestamp,
+                              @QueryParam("step") Integer step,
+                              @QueryParam("isNan") Integer isNan,
+                              @QueryParam("jobName") String jobName,
+                              @QueryParam("id") String id) {
+
+    Metric metric = new Metric();
+    metric.setMetricKey(metricKey);
+    metric.setValue(value);
+    metric.setWorkerIndex(workerIndex);
+    metric.setTimestamp(timestamp);
+    metric.setStep(step);
+    metric.setIsNan(isNan);
+    metric.setJobName(jobName);
+    metric.setId(id);
+
+    LOG.info("listMetric ({})", metric);
+
+    List<Metric> metrics;
+    try {
+      metrics = metricService.selectByPrimaryKeySelective(metric);
+
+    } catch (Exception e) {
+      LOG.error(e.toString());
+
+      return new JsonResponse.Builder<Boolean>(Response.Status.OK).success(true).result(false).build();
```

Comment:
```
return new JsonResponse.Builder<Boolean>(Response.Status.OK).success(true).result(false).build();
```
1. Response.Status.OK: Indicates that the rest request was successful
2. success(true) or success(false): Indicates whether the background processing is normal or abnormal
3. result(false): Indicates that the background processing is normal, then the processing result is returned.
If the background is abnormal, there is no need to return any value. The caller determines whether there is a return value based on the success value.

SO，Please change 
```
return new JsonResponse.Builder<Boolean>(Response.Status.OK).success(true).result(false).build();
```
to
```
return new JsonResponse.Builder<Boolean>(Response.Status.OK).success(false).build();
```